src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -78,23 +85,31 @@ namespace AppInstaller::CLI
         return "https://aka.ms/winget-command-install";
     }
 
-    void InstallCommand::ValidateArgumentsInternal(Execution::Args& execArgs) const
+    void InstallCommand::ValidateArgumentsInternal(Args& execArgs) const
     {
-        if (execArgs.Contains(Execution::Args::Type::Manifest) &&
-            (execArgs.Contains(Execution::Args::Type::Query) ||
-             execArgs.Contains(Execution::Args::Type::Id) ||
-             execArgs.Contains(Execution::Args::Type::Name) ||
-             execArgs.Contains(Execution::Args::Type::Moniker) ||
-             execArgs.Contains(Execution::Args::Type::Version) ||
-             execArgs.Contains(Execution::Args::Type::Channel) ||
-             execArgs.Contains(Execution::Args::Type::Source) ||
-             execArgs.Contains(Execution::Args::Type::Exact)))
+        if (execArgs.Contains(Args::Type::Manifest) &&
+            (execArgs.Contains(Args::Type::Query) ||
+             execArgs.Contains(Args::Type::Id) ||
+             execArgs.Contains(Args::Type::Name) ||
+             execArgs.Contains(Args::Type::Moniker) ||
+             execArgs.Contains(Args::Type::Version) ||
+             execArgs.Contains(Args::Type::Channel) ||
+             execArgs.Contains(Args::Type::Source) ||
+             execArgs.Contains(Args::Type::Exact)))
+        {
+            throw CommandException(Resource::String::BothManifestAndSearchQueryProvided);
+        }
+
+        if (execArgs.Contains(Args::Type::InstallScope))
         {
-            throw CommandException(Resource::String::BothManifestAndSearchQueryProvided, "");
+            if (ConvertToScopeEnum(execArgs.GetArg(Args::Type::InstallScope)) == Manifest::ScopeEnum::Unknown)
+            {
+                throw CommandException(Resource::String::InvalidArgumentValueError, s_ArgumentName_Scope, "user', 'machine");

```

Comment:
>"user', 'machine" [](start = 106, length = 17)

Missing single quote and the leading space?  " 'user', 'machine'"

Reply:
This is because the output of this is value wraps it in single quotes currently.  Maybe I should move it to be a vector of values, then I wouldn't need to do this here.

---
In reply to: [602806765](https://github.com/microsoft/winget-cli/pull/819#discussion_r602806765) [](ancestors = 602806765)